MOBILE_CLIENT/COMMON/_MEDIUM/BIOMETRIC_AUTH_WITHOUT_CRYPTOGRAPHIC_BINDING/meta.json

```
@@ -0,0 +1,22 @@
+{
+  "risk_rating": "high",
```

```suggestion
  "risk_rating": "potentially",
```

Let's ship it as potentially, validate it in scans, see it raising no false positives and THEN change to HIGH.

---

in this case change to _POTENTIALLY as well please

---

if its a temporatu change I dont see the point of moving it to _POTENTIALLY

---

its MEDIUM .. it should be either HIGH or POTENTIALLY